app/services/data_stage/update_staged_schools.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+module DataStage
+  class UpdateStagedSchools < ::BaseService
+    include GiasTypes
+
+    def initialize(school_data_file:, school_links_file:)
+      @school_data_file = school_data_file
+      @school_links_file = school_links_file
+    end
+
+    def call
+      update_schools
+      update_school_links
+    end
+
+  private
+
+    def update_schools
+      if @school_data_file
```

Comment:
When does it not exist? Should we log that it happened (and maybe use early returns to reduce nesting)?

Reply:
Having this in means it would be possible to only update schools or school links if you wanted to (eg dev or testing or manual support).